eidpdata/src/main/java/uk/ac/ucl/eidp/data/jaxb/JaxbSqlAnsi.java

```diff
@@ -59,13 +63,17 @@ protected String buildSetStatement() {
         stm.append(tableType.getName()).append(" SET ");
         
         methodType.getFields().getField().forEach((String f) -> {
-            if (!stm.substring(stm.length() - 4).equals("SET ")) stm.append(", ");
+            if (!stm.substring(stm.length() - 4).equals("SET ")) {
+                stm.append(", ");
```

Comment:
\[Checkstyle\] WARNING: 'if' child have incorrect indentation level 16, expected level should be 12\.